pkg/migration/migration_test.go

```diff
@@ -15,6 +17,7 @@ import (
 
 func TestMain(m *testing.M) {
 	checkpointDumpInterval = 100 * time.Millisecond
+	os.Exit(m.Run())
```

Comment:
This was the fatal mistake. This line was missing.

Reply:
wow , silent killer, good catch.

Reply:
Should we reset the values back to original after the run() method if we want other package tests to use the original values?

Reply:
The other test packages are run concurrently, so they will have to live with these shorter values anyway.